client/state/reader/conversations/test/reducer.js

```diff
@@ -56,6 +57,54 @@ describe( 'reducer', () => {
 			expect( state[ '123-456' ] ).toEqual( 'F' );
 		} );
 
+		test( 'should remove the given site and post from state entirely if followStatus is null', () => {
```

Comment:
instead of "if followStatus is null" it could be "if user is not following the post"